mangum/adapter.py

```diff
@@ -85,36 +92,45 @@ def handler(self, event: dict, context: dict) -> dict:
         return response
 
     def handle_http(self, event: dict, context: dict) -> dict:
-        server, client = get_server_and_client(event)
+        is_http_api = 'multiValueQueryStringParameters' not in event
+        server, client = get_server_and_client(event, is_http_api)
         headers = event.get("headers") or {}
         headers_key_value_pairs = [
             [k.lower().encode(), v.encode()] for k, v in headers.items()
         ]
-        multi_value_query_string_params = event["multiValueQueryStringParameters"]
-        query_string = (
-            urllib.parse.urlencode(multi_value_query_string_params, doseq=True).encode()
-            if multi_value_query_string_params
-            else b""
-        )
+
+        if is_http_api:
+            query_string = event.get('rawQueryString')
+        else:
+            multi_value_query_string_params = event["multiValueQueryStringParameters"]
+            query_string = (
+                    urllib.parse.urlencode(multi_value_query_string_params, doseq=True).encode()
+                    if multi_value_query_string_params
+                    else b""
+            )
+
+        event_path = event['requestContext']['http']['path'] if is_http_api else event["path"]
+        http_method = event['requestContext']['http']['method'] if is_http_api else event["httpMethod"]
+
         scope = {
-            "type": "http",
-            "http_version": "1.1",
-            "method": event["httpMethod"],
-            "headers": headers_key_value_pairs,
-            "path": self.strip_base_path(event["path"]),
-            "raw_path": None,
-            "root_path": "",
-            "scheme": headers.get("X-Forwarded-Proto", "https"),
-            "query_string": query_string,
-            "server": server,
-            "client": client,
-            "asgi": {"version": "3.0"},
-            "aws.event": event,
-            "aws.context": context,
+                "type":         "http",
+                "http_version": "1.1",
+                "method":       http_method,
+                "headers":      headers_key_value_pairs,
+                "path":         self.strip_base_path(event_path),
+                "raw_path":     None,
+                "root_path":    "",
+                "scheme":       headers.get("X-Forwarded-Proto", "https"),
+                "query_string": query_string,
+                "server":       server,
+                "client":       client,
+                "asgi":         {"version": "3.0"},
+                "aws.event":    event,
+                "aws.context":  context,
```

Comment:
Could you please format this using [black](https://github.com/psf/black)?

Reply:
done